### ISSUE TITLE
nfcapd coredump

### ISSUE THREAD

**Issue by shigechika** [phaag/nfdump#259]
Recentry version of nfcapd sometimes coredump.
the following is backtrace.
```
$ rpm -q nfdump
nfdump-1.6.21-1.el7.x86_64
$ sudo gdb /usr/bin/nfcapd /var/crash/core.nfcapd.1145...1605519720
GNU gdb (GDB) Red Hat Enterprise Linux 7.6.1-120.el7
Copyright (C) 2013 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.  Type "show copying"
and "show warranty" for details.
This GDB was configured as "x86_64-redhat-linux-gnu".
For bug reporting instructions, please see:
<http://www.gnu.org/software/gdb/bugs/>...
Reading symbols from /usr/bin/nfcapd...Reading symbols from /usr/lib/debug/usr/bin/nfcapd.debug...done.
done.
[New LWP 1145]
Core was generated by `/usr/bin/nfcapd -y -w -t 60 -D -l /home/work/nf -p 2055 -e -x /usr/bin/nfreplay'.
Program terminated with signal 11, Segmentation fault.
#0  0x00000000004084cf in FlushStdRecords (fs=fs@entry=0xc48020) at collector.c:569
569				AppendToBuffer(fs->nffile, (void *)&(sampler->info), sampler->info.header.size);
(gdb) bt
#0  0x00000000004084cf in FlushStdRecords (fs=fs@entry=0xc48020) at collector.c:569
#1  0x0000000000403ab2 in run (report_seq=<optimized out>, compress=3, time_extension=0x41d937 "%Y%m%d%H%M", use_subdirs=0, t_begin=<optimized out>, twin=60, repeater=0x7ffdadcb3d70, socket=4, receive_packet=0x4026e0 <recvfrom@plt>)
    at nfcapd.c:575
#2  main (argc=<optimized out>, argv=<optimized out>) at nfcapd.c:1221
(gdb) q
```

**Comment by phaag** [comment]
This should be fixed in master repo or release 1.6.22
Thanks for reporting.

**Comment by shigechika** [comment]
I updated to 1.6.22 at 3 days ago.
1.6.21 was crushed every day.
But 1.6.22 does not crash still now.
It seems stable.
Thank you very much.
